data/json/mapgen/sugar_house.json

```diff
@@ -0,0 +1,208 @@
+[
+    {
+    "type" : "item_group",
+    "id" : "sugar_house_items",
+    "items":[
+        [ "syrup", 60 ],
+        [ "sandwich_pbm", 20 ],
+        [ "maple_candy", 90 ],
+        [ "pancakes", 50 ],
+        [ "fruit_pancakes", 30 ],
+        [ "choc_pancakes", 20 ],
+        [ "fruit_waffles", 20 ],
+        [ "choc_waffles", 20 ],
+        [ "pie_maple", 70 ],
+        [ "pork_beans", 15 ],
+        [ "glazed_tenderloin", 5 ],
+        [ "cell_phone", 3 ],
+        [ "glass", 40 ],
+        [ "spoon", 25 ],
+        [ "fork", 25 ],
+        [ "knife_butter", 20 ],
+        [ "knife_steak", 15 ],
+        [ "spork", 5 ],
+        [ "foon", 5 ]
+    ]
+    },{
+    "type" : "item_group",
+    "id" : "sugar_house_kitchen_items",
+    "items":[
+        [ "thermometer", 90 ],
+        [ "family_cookbook", 90 ],
+        [ "maple_sap", 50 ],
+        [ "syrup", 100 ],
+        [ "maple_candy", 10 ],
+        [ "flour", 30 ],
+        [ "milk_powder", 30 ],
+        [ "powder_eggs", 10 ],
+        [ "egg_bird", 10 ],
+        [ "cooking_oil", 20 ],
+        [ "irradiated_apple", 5 ],
+        [ "irradiated_banana", 5 ],
+        [ "irradiated_strawberries", 5 ],
+        [ "chocolate", 20 ],
+        [ "waffleiron", 60 ],
+        [ "bread", 30 ],
+        [ "peanutbutter", 20 ],
+        [ "pot", 80 ]
+    ]
+    },{
+    "type" : "item_group",
+    "id" : "sugar_house_tools",
+    "items":[
+        [ "hammer", 100 ],
+        [ "hand_drill", 100 ],
+        [ "screwdriver", 20 ],
+        [ "screwdriver_set", 10 ],
+        [ "cordless_drill", 70 ],
+        [ "thermometer", 60 ],
+        [ "saw", 90 ],
+        [ "chainsaw_off", 10 ],
+        [ "bucket", 100 ],
+        [ "tree_spile", 100 ],
+        [ "pot", 100 ]
+    ]
+    },{
+    "type" : "item_group",
+    "subtype": "collection",
+    "id": "sugar_house_furnace",
+    "entries":
+    [
+        { "item": "maple_sap", "charges-min": 2, "charges-max": 15, "container-item": "metal_tank" }
+    ]
+    },{
+    "type" : "item_group",
+    "subtype": "collection",
+    "id": "sugar_house_drum",
+    "entries":
+    [
+        { "item": "syrup", "charges-min": 2, "charges-max": 150, "container-item": "55gal_drum" }
```

Comment:
Let's talk about something called "head-space". When storing any nutrient rich broth or syrup, you want to minimize the amount of empty space above the mixture because air is teeming with bacteria. The empty space is "head-space". Here you have 192.5L of air in a container lined an inch deep at the bottom with pure, delicious  carbohydrates. This is a recipe for bacterial contamination. I can see it being alright to either have the drum full to the brim (some may call 4000 charges of syrup too far, I call it not too far enough), or use a smaller container ( such as a jerry can). Other wise this breaks the immersion of every player with rudimentary biology background, especially idiots like me.